static/js/containers/ProfilePage_test.js

```diff
@@ -189,19 +177,20 @@ describe("ProfilePage", function() {
           work_history: [],
         };
         helper.profileGetStub.withArgs(SETTINGS.user.username).returns(Promise.resolve(updatedProfile));
-        return renderComponent(`/profile/${step}`, SUCCESS_ACTIONS).then(([, div]) => {
-          // close all switches
-          if (step === 'personal') {
-            return confirmSaveButtonBehavior(updatedProfile, {div: div}, true);
-          }
-          return confirmSaveButtonBehavior(updatedProfile, {div: div});
+        let actions = step === PERSONAL_STEP ? SUCCESS_ACTIONS.concat(SET_PROGRAM) : SUCCESS_ACTIONS;
+        return renderComponent(`/profile/${step}`, actions).then(([, div]) => {
+          return confirmSaveButtonBehavior(
+            updatedProfile,
+            {div: div},
+            step === PERSONAL_STEP ? [REQUEST_ADD_PROGRAM_ENROLLMENT] : []
```

Comment:
AFAIK the program enrollment API is idempotent, I think that was built that way for this reason, so it should be ok? haha, famous last words.